skimage/_shared/filters.py

```diff
@@ -63,6 +74,15 @@ def gaussian(image, sigma=1, output=None, mode='nearest', cval=0,
         .. versionadded:: 0.19
            ``channel_axis`` was added in 0.19.
 
+        .. warning::
+
+            Automatic detection of the color channel based on the old deprecated
+            `multichannel=None` was broken in version 0.19. In 0.20 this
+            behavior is recovered. The last axis of an `image` with dimensions
+            (M, N, 3) is interpreted as a color channel if `channel_axis` is not
+            set. Starting with 0.21, `channel_axis=None` will be used as the
```

Comment:
I think, given that there's now a "not set" proxy, this deserves further clarification.

Reply:
I tried to clarify it in 529c74a987eea9461e8d7e523cd85e0bf1a17b9d.

Reply:
Thanks @lagru, that looks good to me